pkg/helpers/check/check.go

```diff
@@ -79,3 +78,12 @@ func findResource(list *metav1.APIResourceList, resourceName string) bool {
 	}
 	return false
 }
+
+func IsFeatureEnabled(featureGates []operatorv1.FeatureGate, feature string) bool {
+	for _, fg := range featureGates {
+		if fg.Feature == feature && fg.Mode == operatorv1.FeatureGateModeTypeEnable {
+			return true
```

Comment:
This still does not fit the add-on manager case, because it is enabled by default.
so there are two cases:
- enabled by default(eg. addon-manager): as long as the feature gate flag is not explicitly set to false(including the feature gate config is not set), we should output the info
- disabled by default(eg. work-controller): only output the info when the feature gate flag is explicitly set to true